include/evmc/evmc.hpp

```diff
@@ -408,12 +408,12 @@ class VM
     }
 
 private:
-    evmc_instance* m_instance = nullptr;
+    evmc_vm* m_instance = nullptr;
```

Comment:
I have left the name `m_instance` because `m_vm` would be worse. This can also be `m_handle` (as in Go) or `m_vm_ptr`.